pages/linux/i7z.md

```diff
@@ -0,0 +1,7 @@
+# i7z
+
+> A CPU (only i3, i5 and i7) realtime reporting tool.
```

Comment:
Perhaps better to add "Intel" for extra clarity ?

Reply:
Yeah, I've considered that but there is no other CPU models called i3/5/7. So I'm not sure if I should use description 
`An Intel CPU (only i3, i5 and i7) realtime reporting tool.`